### ISSUE TITLE
Display camera image on a dashboard

### ISSUE THREAD

**Issue by DeltaFox0018** [myhomeiot/DahuaVTO#24]
is it possible to display the live stream captured by the camera present in the VTO in a dashboard inside a card?

**Comment by myhomeiot** [comment]
Yes, but you should use Generic Camera (or ONVIF) integration. For still image url use:
`http://192.168.1.x/cgi-bin/snapshot.cgi?channel=1`
For stream url:
`rtsp://admin:password@192.168.1.x:554/cam/realmonitor?channel=1&subtype=0`
Authentication: `digest`
`Username and password` field - specify correct user name and password for VTO web GUI, in stream url replace password with correct password and 192.168.1.x with correct IP.

**Comment by DeltaFox0018** [comment]
Ok perfect, I just implemented it and it works
Thanks
